prime-router/docs/observability/azure-events.md

```diff
@@ -93,6 +93,24 @@ Many `ReportRouteEvent` can correspond to a `ReportAcceptedEvent` and can be "jo
 - topic
     - The Topic of the report
 
+### ReportReceiverSelectedEvent
+This event is emitted during the DestinationFilter routing step when a report is queued for receiver filter evaluation.
+Many `ReportReceiverSelectedEvent` can correspond to a `ReportAcceptedEvent` and can be "joined" on
+`ReportAcceptedEvent.reportId == ReportReceiverSelectedEvent.parentReportId`.
+
+- parentReportId
```

Comment:
Would `submittedReportID` differ from existing `parentReportId`? (i.e. rename the variable for this event and add it to others?).

Reply:
Yes it would. This is my understanding:
reportID: the report ID of the new report we create in this step
parentID: the reportID that is passed in this step's queue message. In other words, the report ID of the report that was created in the convert step
submittedID: the reportID that was created in the receive step when sender uploaded the report.